src/sklearn_evaluation/plot/grid_search.py

```diff
@@ -22,6 +23,39 @@
 )
 
 
+def _validate_change_input(change):
+    if change is None:
+        raise ValueError(
+            (
+                "change can't be None, you need to select at least"
+                " one value to make the plot."
+            )
+        )
+
+    to_validate = change
+
+    if isinstance(change, str):
+        to_validate = [change]
+
+    for input in to_validate:
+        keys(
+            valid=[
+                "n_estimators",
+                "criterion",
+                "max_depth",
+                "max_features",
+                "min_samples_split",
+                "min_samples_leaf",
```

Comment:
the valid values depend on the input data. these parameters are only valid for random forest, but the grid search plot accepts input data from any model  (which have different parameter names); so the validation should be based on that. here's where we extract them for a [single parameter plot](https://github.com/ploomber/sklearn-evaluation/blob/682d073ca1b1d1933072af6812f8f700bd2df268/src/sklearn_evaluation/plot/grid_search.py#L114), a similar thing can be done for a two-parameter plot